execution_engine/src/core/runtime/externals.rs

```diff
@@ -992,10 +996,10 @@ where
                     _,
                 ) = Args::parse(args)?;
                 self.charge_host_function_call(
-                    &host_function_costs.dictionary_get,
+                    &host_function_costs.read_value_local,
```

Comment:
I see there's a commit `Revert backwards incompatible changes` where you reverted your approach of `s/local/dictionary/g` for several field and type names in `HostFunctionCosts` among other places. Is this due to `serde`'s `Serialize` and `Deserialize` traits? If so, could we leverage `serde`'s `#[serde(alias = "...")]` attribute here?

e.g.
```
#[derive(Copy, Clone, PartialEq, Eq, Serialize, Deserialize, Debug, DataSize)]
pub struct HostFunctionCosts {
...
    #[serde(alias = "read_local")]
    pub dictionary_get: HostFunction<[Cost; 4]>,
...
}
```
This would preserve the ability to deserialize from the old name, while moving the changing the field name to reflect the `dictionary` name.

Reply:
Fair point, changed